### PR TITLE
Astro adding e2e tests

### PR DESCRIPTION
**Pull request by omar-h-omar**

### CODE REVIEW

File: .github/workflows/playwright.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+          sudo /etc/init.d/mysql start
+          mysql -e 'CREATE DATABASE test;' -uroot -proot
+          mysql -e 'SET GLOBAL SQL_MODE='ALLOW_INVALID_DATES';' -uroot -proot
+      - name: Run Playwright tests
+        run: |
+          cd readers-frontend
+          npm run test
+        env:
+          DATABASE_URL: "mysql://root:root@localhost:3306/test"
+          fileStoreUrl: "https://bbcdn.nouse.co.uk/file"
```
Comment (joe2k01) [inline .github/workflows/playwright.yml:38]: Missing https://github.com/yorknouse/website/blob/astro/readers-frontend/src/components/utils/articles.ts#L56 ?

Comment (omar-h-omar) [reply .github/workflows/playwright.yml:38]: Good catch! I've added it now

File: readers-frontend/src/components/utils/articles.ts [hunk 2]
```diff
@@ -3,55 +3,68 @@ import prisma from "../../prisma";
...
+  size: "tiny" | "small" | "medium" | "large" | false = "large"
+): Promise<string> => {
+  if (!article.articles_displayImages) {
+    return (
+      import.meta.env.fileStoreUrl + "/nouseSiteAssets/imageArchive-comp.jpg"
+    );
+  } else if (Number(article.articles_thumbnail)) {
+    return await s3URL(Number(article.articles_thumbnail), size);
+  } else
+    return import.meta.env.ARCHIVEFILESTOREURL + article.articles_thumbnail;
```
Comment (joe2k01) [inline readers-frontend/src/components/utils/articles.ts:69]: The env variable is lowercase

**Review by joe2k01 — CHANGES_REQUESTED** [review]

File: readers-frontend/src/components/utils/articles.ts (hunk 2 above)
Comment (omar-h-omar) [reply readers-frontend/src/components/utils/articles.ts]: ahh I see I thought you meant the env in the tests. This was there from before this ticket

**Review by joe2k01 — APPROVED** [review]

**Pull request merged**